src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -18,6 +18,20 @@ public static class ConfigurationKeys
         /// <seealso cref="TracerSettings.Environment"/>
         public const string Environment = "DD_ENV";
 
+        /// <summary>
+        /// Configuration key for the application's version. Sets the "version" tag on every <see cref="Span"/>.
+        /// </summary>
+        /// <seealso cref="TracerSettings.Version"/>
+        public const string Version = "DD_VERSION";
```

Comment:
Can we call this `ServiceVersion` to avoid confusion?

Reply:
As long as this doesn't cause confusion with the OpenTracing `service.version`, I think this is fine.

Reply:
Done in 1c1b222185b9dbeacf6849d99d8d5181dfe888c9